html2asketch/symbolInstance.js

```diff
@@ -0,0 +1,39 @@
+import Base from './base';
```

Comment:
can we have some tests for it?

Reply:
Absolutely! I just added a few basic tests – they should be sufficient to make sure our branches work together :)